recipes/bioconductor-dnacopy/1.48.0/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: bioconductor-dnacopy
+  version: 1.48.0
+source:
+  fn: DNAcopy_1.48.0.tar.gz
+  url: https://bioarchive.galaxyproject.org/DNAcopy_1.48.0.tar.gz
+  sha256: 4094d45c761d2494912f07dc6f8f70f6c37c1831220bebc28404a9d95203fbbf
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - r-base
+    - gcc
+  run:
+    - r-base
```

Comment:
What about here? Also "libgcc     # [not win]"? I cannot find a pattern in the existing YAMLs. Sometimes its with libgcc in the run-block, sometimes with # [not win]. So what is the rule?

The same for llvm and OSX?